### PR TITLE
startOf function is accepting number as weekday from 0(Sun) to 6(Sat)

### LINKED ISSUE #17: Fix startOf function for isoWeek
The startOf function is accepting a boolean for Sunday(false) and Monday(true).

Instead it should get the the week day (number) between 0 (Sunday) and 6(Saturday) in order to be more flexible to provide the startOf the week based on the parameter.

See https://github.com/chartjs/chartjs-adapter-luxon/issues/23

### PR DESCRIPTION
**Pull request by stockiNail**
Fixes #17

### CODE REVIEW

**Review by etimberg — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -47,7 +47,8 @@ Chart._adapters._date.override(typeof moment === 'function' ? {
 	startOf: function(time, unit, weekday) {
 		time = moment(time);
 		if (unit === 'isoWeek') {
-			return time.isoWeekday(weekday).valueOf();
+			weekday = Math.trunc(Math.min(Math.max(0, weekday), 6));
+			return time.subtract((time.isoWeekday() < weekday ? 7 : 0) + time.isoWeekday() - weekday, 'days').startOf('day').valueOf();
```
Comment (kurkle) [inline src/index.js:51]: why not `time.isoWeekday(weekday).startOf('day').valueOf();`?

Comment (stockiNail) [reply src/index.js]: changed as proposed. Tested it sounds working correctly!

**Review by kurkle — APPROVED** [review]

**Review by etimberg — APPROVED** [review]

**Comment by etimberg** [comment]
Looks like there are some lint issues. Once those are resolved, I'll merge

**Comment by stockiNail** [comment]
> 
> 
> Looks like there are some lint issues. Once those are resolved, I'll merge

Lint issues has been removed.

Thanks a lot!

**Review by etimberg — APPROVED** [review]

**Pull request merged**